suite-native/module-send/src/sendOutputsFormSchema.ts

```diff
@@ -133,13 +181,39 @@ export const sendOutputsFormValidationSchema = yup.object({
                     ),
                 fiat: yup.string(),
                 token: yup.string().required().nullable(),
-                // TODO: other validations have to be added in the following PRs
-                //       e.g. check if the amount is not higher than XRP reserve
             }),
         )
         .required(),
+    rippleDestinationTag: yup
+        .string()
+        .optional()
+        .matches(
+            /^\d*$/,
+            'You can only use positive numbers for the destination tag. fdsfds dsf dsfsf dsf dsf sdf dsf dsf .',
```

Comment:
```suggestion
            'You can only use positive numbers for the destination tag.',
```

Reply:
Damn, I was testing a long error message that can not fit in the screen width and forgot it there. Fixed 8839f5a337